core/metrics-core-service/src/main/java/org/hawkular/metrics/core/dropwizard/MetricNameService.java

```diff
@@ -33,28 +33,45 @@
  */
 public class MetricNameService implements MetricFilter {
 
-    public static final String TENANT_ID = "admin";
 
     private static final String separator = ":";
 
-    private String hostname;
+    private final String adminTenant;
+    private final String hostname;
 
     public MetricNameService() {
         try {
             hostname = InetAddress.getLocalHost().getHostName();
         } catch (UnknownHostException e) {
             throw new RuntimeException("Failed to get hostname", e);
         }
+
+        this.adminTenant = "admin";
```

Comment:
Sorry, I misunderstood the previous comments and diff. The no-arg constructor gets called if neither the ```hawkular.metrics.hostname``` system property nor the ```HOSTNAME``` environment variable are set.